ledger/src/blockstore_processor.rs

```diff
@@ -74,7 +74,7 @@ pub struct BlockCostCapacityMeter {
 
 impl Default for BlockCostCapacityMeter {
     fn default() -> Self {
-        BlockCostCapacityMeter::new(MAX_BLOCK_UNITS)
+        BlockCostCapacityMeter::new((MAX_BLOCK_UNITS as f64 * 1.10) as u64)
```

Comment:
not that it matters if replay_stage doesnt have buffer. But can avoid floats by `(UNITS * 10u64) / 100u64`